app/components/side-bar.js

```diff
@@ -58,3 +58,22 @@ export default class SideBar extends Component {
     }
   }
 }
+
+$(function() {
+
+  $(window).bind('scroll', function() {
+    const menuPosition = $('#col-content').offset().top;
+
+    if ($(window).scrollTop() > menuPosition) {
+      $('#public-event-content').addClass('fixed');
+    } else {
+      $('#public-event-content').removeClass('fixed');
+    }
+    const ticketPosition = $('#tickets').offset().top;
+    if ($(window).scrollTop() > ticketPosition) {
+      $('#public-event-content .text').html('<i class="mr-4  ticket icon"></i>Tickets');
+    } else {
+      $('#public-event-content .text').html('<i class="mr-4  info icon"></i>Info');
```

Comment:
above code is cool, but why you need to do this ?

Reply:
This is done to fix the position of the menu bar at the top on scroll and from line no 72 to 76 as above mentioned by using this I'm changing the default displayed text of the menu bar to "Tickets" when user scroll down and reaches to the ticket section, because this is what the need as described in the problem!